scss/main.scss

```diff
@@ -287,23 +287,55 @@ h1,h2,h3 {
   margin-bottom: 2rem;
   padding-left: 2rem;
   padding-right: 2rem;
+  p.text-center {
+    margin-bottom: 48px;
+  }
 }
 
 #meetupLink {
   color: $cfa-blue;
 }
 
+.meetup-content {
+  margin-bottom: 1.5rem;
+  > * {
+  margin-bottom: 1rem;
+  }
+}
+
+
+
 .rsvp-stuff {
   margin-top: 1rem;
+  p {
+    font-family: $font-family-alt;
+    font-weight: 500;
+    color: $darker-grey;
+    font-size: .875rem; 
+  }
 }
 
 .attendees {
   border-bottom: 2rem solid transparent;
 }
 
 .attend-button {
+  font-family: $font-family-alt;
+  font-weight: 500;
+  color: $darker-grey; 
+  font-size: .875rem;
   border-bottom: 1rem solid transparent;
   padding-left: 5%;
```

Comment:
This padding is making the button slightly off-center in the mobile view:

<img width="547" alt="Screen Shot 2021-02-10 at 9 29 05 PM" src="https://user-images.githubusercontent.com/9326234/107600846-8e5e5c00-6bea-11eb-94c7-29c752f2b566.png">